script/CheckSecurityConfigs.s.sol

```diff
@@ -0,0 +1,192 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {console2} from "forge-std/console2.sol";
+import {Script} from "forge-std/Script.sol";
+import {VmSafe} from "forge-std/Vm.sol";
+
+/**
+ * @title CheckSecurityConfigs
+ * @notice A script to check security configurations of an OP Chain,
+ *            such as upgrade key holder, challenger and guadian designations.
+ *         The usage is as follows:
+ *         $ forge script CheckSecurityConfigs \
+ *             --rpc-url $MAINNET_RPC_URL
+ */
+
+contract CheckSecurityConfigs is Script {
+    struct ProtocolAddresses {
+        // Protocol contracts
+        address AddressManager;
+        address L1CrossDomainMessengerProxy;
+        address L1ERC721BridgeProxy;
+        address L1StandardBridgeProxy;
+        address L2OutputOracleProxy;
+        address OptimismMintableERC20FactoryProxy;
+        address OptimismPortalProxy;
+        address ProxyAdmin;
+        address SystemConfigProxy;
+        // Privileged roles
+        address Challenger;
+        address Guardian;
+        address ProxyAdminOwner;
+        address SystemConfigOwner;
+    }
+
+    bool internal hasErrors;
+
+    /**
+     * @notice The entrypoint function.
+     */
+    function run() external {
+        VmSafe.DirEntry[] memory addressesJsonEntries = vm.readDir("superchain/extra/addresses/mainnet");
+        hasErrors = false;
+        for (uint256 i = 0; i < addressesJsonEntries.length; i++) {
+            require(bytes(addressesJsonEntries[i].errorMessage).length == 0, addressesJsonEntries[i].errorMessage);
+            runOnSingleFile(addressesJsonEntries[i].path);
+        }
+        require(!hasErrors, "Errors occurred: See logs above for more info");
+    }
+
+    function findInDirectory(string memory directory, string memory nameRegex) internal returns (string[] memory) {
+        string[] memory inputs = new string[](4);
+        inputs[0] = "find";
+        inputs[1] = directory;
+        inputs[2] = "-name";
+        inputs[3] = nameRegex;
+        string memory files = string(vm.ffi(inputs));
```

Comment:
FFI is disabled by default in forge for security reasons, so we either need to add `ffi = true` to the `foundry.toml` file OR add a justfile that uses the `--ffi` CLI flag.

Though, it seems this function is never actually used, so it seems we can just delete this avoid using ffi for now